qwed_finance/query_guard.py

```
@@ -153,6 +153,11 @@ def _analyze_with_sqlglot(self, sql_query: str) -> dict:
                 result["query_type"] = "CREATE"
                 result["violations"].append("DDL detected: CREATE statement")
             
+            # Check for mutations in subqueries
+            for node in parsed.find_all(exp.Insert, exp.Update, exp.Delete, exp.Drop, exp.Create):
+                if node == parsed: continue # Already handled top-level
```

Ruff (0.14.14)

[error] Multiple statements on one line (colon)

(E701)